fuzzers/main.py

```diff
@@ -655,17 +657,17 @@ def signal(self, sig: int) -> None:
 T = typing.TypeVar('T')  # pylint: disable=invalid-name
 
 
-def random_weighted(array: typing.List[T], num: int, name: str) -> typing.List[T]:
+def random_weighted(array: typing.List[T], name: str) -> T:
     """Pick `num` random items from `array`, logging that as `name`
```

Comment:
Docstring is no longer accurate. Also in Args section.

Reply:
Fixed